### DISCUSSION TITLE
How do i should import UI library in exported project?

### DISCUSSION THREAD

**Discussion by CREAsTIVE** [AssetRipper/AssetRipper#1227] [Q&A]
When i try to decompile _(slime rancher)_ and launch it i got many errors where `UI` and `EventSystems` just doesn't exist. I m even didn't have UI elements when i try to create new `GameObject` (i a'm already try to put `GameAssemblies` inside `Library` folder)

**Comment by CREAsTIVE** [comment]
ScriptAssemblies automatic clear self when i try to launch project

**Comment by CREAsTIVE** [comment]
I copy libraries from `package-lock.json` inside new project to same file inside decompiled project and this fix my issue (but some libraries still wont load)

**Comment by CREAsTIVE** [comment]
`TMPro.TMP_Text` component just won't import idk

(there should ve TMP_Text): 
![image](https://github.com/AssetRipper/AssetRipper/assets/62616308/d121172a-92c6-4575-a50f-f3343190bd90)

**Comment by thebxzr** [comment]
I had this same problem before, you have to go into Package Manager in Unity and add the package from there, and delete in in the Scripts folder.
